internal/collector/collector_test.go

```diff
@@ -520,13 +556,14 @@ var _ = Describe("Factory#Deployment#NewPodSpec", func() {
 					ReadOnly:  true,
 					MountPath: constants.TrustedCABundleMountDir,
 				}))
-			})
-		})
-
-		Context("and mounting volumes", func() {
-			It("should not mount host path volumes", func() {
-				Expect(podSpec.Volumes).NotTo(ContainElement(v1.Volume{Name: sourcePodsName, VolumeSource: v1.VolumeSource{HostPath: &v1.HostPathVolumeSource{Path: sourcePodsPath}}}))
-				Expect(podSpec.Volumes).To(HaveLen(6))
+				Expect(podSpec.Volumes).To(ContainElement(v1.Volume{
```

Comment:
There is a custom matcher for this though in review I am unable to attest to its value now